sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/CommitLog.scala

```diff
@@ -51,14 +51,18 @@ class CommitLog(sparkSession: SparkSession, path: String)
 
   import CommitLog._
 
+  private val VERSION = SQLConf.get.stateStoreCheckpointFormatVersion
```

Comment:
What's the purpose of moving this variable to here?

Reply:
I think moving it here is correct since the conf can be changed per session. Can we use `sparkSession.conf` instead though, not SQLConf.get?